### PR TITLE
Gh 2268 unable to apply BiFunction to view

### PR DESCRIPTION
**Pull request by m29827**

### CODE REVIEW

File: core/data/src/test/java/uk/gov/gchq/gaffer/data/element/function/ToElementTupleTest.java [hunk 1]
```diff
@@ -71,6 +71,16 @@ protected ToElementTuple getInstance() {
...
     }
 
+    @Override
+    protected Class[] getExpectedSignatureInputClasses() {
+        return new Class[]{Element.class};
+    }
+
+    @Override
+    protected Class[] getExpectedSignatureOutputClasses() {
+        return new Class[]{};
```
Comment (d47853) [inline core/data/src/test/java/uk/gov/gchq/gaffer/data/element/function/ToElementTupleTest.java:81]: I would have expected this to be an ElementTuple

File: core/data/src/test/java/uk/gov/gchq/gaffer/data/element/function/ToPropertiesTupleTest.java [hunk 2]
```diff
@@ -65,6 +65,16 @@ protected ToPropertiesTuple getInstance() {
...
     }
 
+    @Override
+    protected Class[] getExpectedSignatureInputClasses() {
+        return new Class[]{Properties.class};
+    }
+
+    @Override
+    protected Class[] getExpectedSignatureOutputClasses() {
+        return new Class[]{};
```
Comment (d47853) [inline core/data/src/test/java/uk/gov/gchq/gaffer/data/element/function/ToPropertiesTupleTest.java:75]: I would have expected this to be a PropertiesTuple

File: core/data/src/test/java/uk/gov/gchq/gaffer/data/element/function/ToElementTupleTest.java (hunk 1 above)
Comment (m29827) [reply core/data/src/test/java/uk/gov/gchq/gaffer/data/element/function/ToElementTupleTest.java]: Now fixed

File: core/data/src/test/java/uk/gov/gchq/gaffer/data/element/function/ToPropertiesTupleTest.java (hunk 2 above)
Comment (m29827) [reply core/data/src/test/java/uk/gov/gchq/gaffer/data/element/function/ToPropertiesTupleTest.java]: Now fixed

**Pull request merged**